src/routes/blog/post/the-appwrite-network/+page.markdoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: The Appwrite Network announcement
+description: Announcing more regions for Cloud, and many more to come with the introduction of the Appwrite Network
+date: 2025-04-15
+cover: /images/blog/the-appwrite-network/network-cover-image.png
+timeToRead: 05
+author: christy-jacob
+category: product
+featured: true
+callToAction: true
+---
+
+# Announcing the Appwrite Network: Appwrite’s vision for a global cloud infrastructure
+
+We are happy to announce the launch of the Appwrite Network, a network of cloud regions and edge locations (edges) to improve Appwrite Cloud availability, performance, and compliance with local regulations. This will provide Appwrite developers and teams with the best tools and infrastructure to build, deploy, and scale your applications. This brings us closer to the Appwrite mission of making software development more accessible and enjoyable for all developers.
+
+Starting today, all Pro users have access to 3 regions of choice, including our two new regions in **New York City** ('NYC'), and **Sydney** ('SYD') that are joining our first region in **Frankfurt** ('FRA'). In the next few weeks, this will be available to all Cloud users, with more regions to come in the next few weeks.
```

Comment:
```suggestion
Starting today, all Pro users have access to 3 regions of choice, including our two new regions in **New York City** ('NYC'), and **Sydney** ('SYD') that are joining our first region in **Frankfurt** ('FRA'). In the next few weeks, this will be available to all Cloud users, with more regions to come.
```